e2e/scenario_test.go

```diff
@@ -1187,3 +1187,42 @@ func Test_ubuntu1804imdsrestriction_mangletable(t *testing.T) {
 		},
 	})
 }
+
+func Test_Ubuntu2204MessageOfTheDay(t *testing.T) {
+	RunScenario(t, &Scenario{
+		Tags: Tags{
+			ServerTLSBootstrapping: true,
```

Comment:
we can remove this tag, this isn't a server TLS bootstrapping scenario

Reply:
ops, copy & paste error